src/libs/actions/BankAccounts.ts

```diff
@@ -427,7 +427,7 @@ function connectBankAccountManually(bankAccountID: number, accountNumber?: strin
 /**
  * Verify the user's identity via Onfido
  */
-function verifyIdentityForBankAccount(bankAccountID: number, onfidoData: OnfidoData) {
+function verifyIdentityForBankAccount(bankAccountID: number, onfidoData: Record<string, unknown>) {
```

Comment:
NABish but this seems sus – why are we using `Record<string, unknown>` here?

Reply:
Let's do it but in a different PR, [here](https://github.com/Expensify/App/pull/31378#issuecomment-1944379446) the type for onfidoData is added